src/component/Home.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import { EntryService } from "../service/EntryService";
+
+interface MyState {
+    loading: boolean;
+    message: string;
+}
+
+class Home extends React.Component<{},MyState> {
+    state: MyState = {
+      loading: true,
+      message: ''
+    };
+    private entryService: EntryService;
+    constructor(props: any) {
+        super(props);
+        this.entryService = new EntryService();
+        this.refreshEntry = this.refreshEntry.bind(this);
+    }
+
+    componentDidMount() {
+        this.refreshEntry();
+    }
+
+    render() {
+      return (
+        <div>
+          {this.state.loading ? 'Loading....' : this.state.message} <br />
+          <button onClick={this.refreshEntry}>Refresh</button>
```

Comment:
Ah, hiervoor doe je in de constructor `this.refreshEntry = this.refreshEntry.bind(this);`. In mijn opinie is het dan beter om hier ofwel:
`onClick={() => this.refreshEntry()}`
of
`onClick={this.refreshEntry.bind(this)}`
te doen